src/kernel-install/kernel-install.in

```diff
@@ -75,7 +75,7 @@ fi
 export KERNEL_INSTALL_VERBOSE=0
 if [ "$1" = "--verbose" ] || [ "$1" = "-v" ]; then
     shift
-    KERNEL_INSTALL_VERBOSE=1
+    export KERNEL_INSTALL_VERBOSE=1
```

Comment:
this was dropped in 76b1274a5cb54acaa4a0f0c2e570d751f9067c06

Hmm, isn't the "export-and-set-in-one" thing a bashism? Shouldn't this be done in two lines?

/cc @nabijaczleweli

Reply:
It's not exactly bashism, it's even allowed by POSIX, but some shells don't support it. However, I can't recall any shell we "support" that doesn't support this syntax.

Reply:
I ran some tests on the more exotic/baseline shells I could think of (pdksh/mksh/yash/mrsh/NetBSD sh) and every one I tried supported this, so I don't think this is an issue for any shell in general.